### ISSUE TITLE
Notepad++ 8.3.2 doesn't work with JSON Viewer plugin 1.4.0

### ISSUE THREAD

**Issue by nodeticswww** [notepad-plus-plus/notepad-plus-plus#11380]
### Description of the Issue
Notepad++ 8.3.2 (32 bit or 64 bit) doesn't work with JSON Viewer plugin 1.4.0. Notepad++ doesn't start.

### Steps to Reproduce the Issue

1. Install earlier version of Notepad++ and JSON Viewer plugin 1.4.0
2. Uninstall the earlier version 
3. Install Nodepad++ 8.3.2
4. Start Notepad++

### Expected Behavior
Notepad++ starts normally.

### Actual Behavior
Notepad++ doesn't start at all (and there's no error message either).

### Debug Information
Notepad 8.2.1 works fine.

**Comment by donho** [comment]
@nodeticswww 
I cannot reproduce at all with my v8.3.3. It could be the other reason than plugin issue.
Could you try v8.3.3 : 
https://community.notepad-plus-plus.org/topic/22704/notepad-v8-3-3-release-candidate

**Comment by ArkadiuszMichalski** [comment]
@nodeticswww Are you sure the problem is [NPPJSONViewer plugin](https://github.com/kapilratnani/JSON-Viewer)? For me works fine:

![image](https://user-images.githubusercontent.com/2730894/159580610-2868b2f0-e6e5-4805-9931-b1d9f4fd496a.png)

**Comment by rdipardo** [comment]
@ArkadiuszMichalski,
> For me works fine.

Did you try the 64-bit version?

Not to look for trouble, but some bit rot is very likely in this case, as JSON-Viewer has [no active maintainer][1], and the [Scintilla interface][2] hasn't been updated since v3.5.6:

![Sci_PostionCR_compared](https://user-images.githubusercontent.com/59004801/159589362-ccfcf10e-c72c-4ec8-af6f-b18f71db0a34.png)


![SCi_Notify_hdr_compared](https://user-images.githubusercontent.com/59004801/159589332-86651531-8d1c-48fb-9bb9-6a554d1e614a.png)



[1]: https://github.com/kapilratnani/JSON-Viewer/issues/121
[2]: https://github.com/kapilratnani/JSON-Viewer/blob/28a18f421d70e995071fbd97281aba51af631317/NppJSONViewer/Scintilla.h#L1065-L1068

**Comment by ArkadiuszMichalski** [comment]
>@rdipardo  Did you try the 64-bit version?

Yes, check also 64-bit and it works. Also notice that opener mention that both versions don't work for him.

**Comment by rdipardo** [comment]
I can't reproduce it, either, and the code does nothing with the deadly APIs (except define them):

    $ git grep -niE "Sci_(PostionCR|TextToFind|GETLENGTH|GETTEXTRANGE)"

    NppJSONViewer/Scintilla.h:402:#define SCI_GETTEXTRANGE 2162
    NppJSONViewer/Scintilla.h:1075:struct Sci_TextToFind {
    NppJSONViewer/Scintilla.h:1083:#define TextToFind Sci_TextToFind


Some ideas:

- the JSON file was saved with no line breaks and caused a repeat of #11427 or #10407

  _But then, why is 8.2.1 reportedly immune, if #10407 affected 8.1.3 (32-bit)?_

- the JSON file was "close to 650MB"[^1] and caused a repeat of #11103

  _But then, release builds are never compiled with GCC_

- some other plugin is at fault

  _No Debug Info?_ (*)
  
  
---
(*) I suggest this project start using the [newer issue template setup][1]; especially the option to _enforce the template!_

~~~yaml
# .github/ISSUE_TEMPLATE/config.yml
blank_issues_enabled: false
~~~  

[^1]: Quoting https://github.com/notepad-plus-plus/notepad-plus-plus/issues/11103#issuecomment-1024907213

[1]: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

**Comment by ArkadiuszMichalski** [comment]
@nodeticswww You still have this problem?

**Comment by nodeticswww** [comment]
As mentioned, Notepad++ 8.2.1 works fine so I'm using that one instead of the latest version.

**Comment by ArkadiuszMichalski** [comment]
>As mentioned, Notepad++ 8.2.1 works fine so I'm using that one instead of the latest version.

This does not help determine what the problem is. Please check if problem still exist on latest Notepad++ portable 8.4.4.
